docs/03-extensions/external-login-handlers/index.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: "External Login Handlers"
+description:
+  "This extension allow users to sign-ing and sign-up to your website using
+  external providers like Google or Facebook."
+---
+
+<p>{frontMatter.description}</p>
+
+## Compatibility
+
+:::danger
+
+At the time of writing, we only support this feature for Magento2. Magento1
+support will come soon.
+
+:::
```

Comment:
**issue:** I think it's better to have a dedicated page for this. Like for providers.

**suggestion:** create a page for External Login adapters. It could explain what is an adapter (i.e: allowing users to connect to this system) and the core extensions supporting this feature.

It can also contain a section explaining how to register an adapter without entering into further details (with a callout indicating that it's an advanced use case that will better have to be discussed with our support team before taking actions).

This section would link to this new page.

Reply:
Fixed with your suggestion